src/codegen/utilities/attribute_helpers.py

```diff
@@ -0,0 +1,15 @@
+import logging
+from collections import namedtuple
+from codegen.utilities.properties.attribute import attribute
+
+_logger = logging.getLogger(__name__)
+_logger.addHandler(logging.NullHandler())
+
+def get_attributes(metadata, className):
+    attributes_metadata = []
```

Comment:
This method converts the metadata to a list of `attribute` objects that are part of the specified class.